src/App.vue

```diff
@@ -255,6 +256,8 @@ export default {
 
 		this.onResize()
 
+		setGlobalToastOptions({ selector: this.$store.getters.getMainContainerSelector().slice(1) })
+
```

Comment:
I haven't tested yet, but it seems, it should be watch. Otherwise, it is set once and doesn't update global selector with the getMainContainerSelector update.